pyproject.toml

```diff
@@ -25,6 +25,9 @@ python = "^3.6"
 "ruamel.yaml" = "^0.16.10"
 typed-argument-parser = "^1.6.3"
 pynvim = "^0.4.2"
+pyright = "^0.0.7"
+flake8 = "^3.9.2"
+typing-extensions = "^3.10.0"
```

Comment:
Move to dev-dependencies.

Reply:
typing-extensions is not dev-dependency.

I will move pyright and flake8 immediately :V